schema/types/Answer.js

```diff
@@ -0,0 +1,40 @@
+const { GraphQLObjectType, GraphQLString, GraphQLBoolean, GraphQLInt } = require("graphql");
+
+module.exports = new GraphQLObjectType({
+  name : "Answer",
+  description : "An answer",
+
+  fields : () => ({
+    id : {
+      type: GraphQLInt
```

Comment:
Just a small point but I wonder if the `Id` fields should be of type `GraphQLID` for better semantics?